GrandCentralBoard/Widgets/Harvest/FetchAccessTokenRequest.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FetchAccessTokenRequest.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-14.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+import GrandCentralBoardCore
+
+
+class FetchAccessTokenRequest {
+    let account: String
+    let refreshCredentials: TokenRefreshCredentials
+    let downloader: NetworkRequestManager
+
+    init(account: String, refreshCredentials: TokenRefreshCredentials, downloader: NetworkRequestManager) {
+        self.account = account
+        self.refreshCredentials = refreshCredentials
+        self.downloader = downloader
+    }
+
+    func fetch(completion: (Result<AccessToken>) -> Void) {
```

Comment:
I would prefer to see what this is fetching in the method's name. `fetchAccessToken`?